backend/spec/requests/answers_surveys_request_spec.rb

```diff
@@ -31,6 +31,7 @@
             'name' => survey.name,
             'description' => survey.description,
             'survey_subject_id' => survey_subject.id,
+            'answered_questions_quantity' => 0,
```

Comment:
And if you do like this? what do you think?
```suggestion
            'answered_questions_quantity' => survey.answered_questions_quantity,
```